### ISSUE TITLE
DuinoCoin@RPI, install python3-pip

### ISSUE THREAD

**Issue by PSLLSP** [duino-coin/duino-coin#1689]
Small issue with documentation for DuinoCoin miner at RaspberyPI (and mayby others), page https://duinocoin.com/getting-started.html

page has instruction:

Next, install python and git: `sudo apt install python3 python3-dev git`

`pip` package is missing in the list, it should be:

Next, install python and git: `sudo apt install python3 python3-dev python3-pip git`

---

Instruction

`sudo apt update` and then `sudo apt upgrade -y`

could be written as 

`sudo apt update && sudo apt upgrade -y`

---

Anyway, PC_miner.py has some issue

```
$ python3 --version
Python 3.11.2

$ python3 PC_Miner.py 
Colorama is not installed. Miner will try to automatically install it If it fails, please manually execute python3 -m pip install colorama
WARNING: pip is being invoked by an old script wrapper. This will fail in a future version of pip.
Please see https://github.com/pypa/pip/issues/5599 for advice on fixing the underlying issue.
To avoid this problem you can invoke Python with '-m pip' instead of running pip directly.
error: externally-managed-environment

× This environment is externally managed
╰─> To install Python packages system-wide, try apt install
    python3-xyz, where xyz is the package you are trying to
    install.
    
    If you wish to install a non-Debian-packaged Python package,
    create a virtual environment using python3 -m venv path/to/venv.
    Then use path/to/venv/bin/python and path/to/venv/bin/pip. Make
    sure you have python3-full installed.
    
    For more information visit http://rptl.io/venv

note: If you believe this is a mistake, please contact your Python installation or OS distribution provider. You can override this, at the risk of breaking your Python installation or OS, by passing --break-system-packages.
hint: See PEP 668 for the detailed specification.
```

```
$ python3 -m pip install colorama
error: externally-managed-environment

× This environment is externally managed
╰─> To install Python packages system-wide, try apt install
    python3-xyz, where xyz is the package you are trying to
    install.
    
    If you wish to install a non-Debian-packaged Python package,
    create a virtual environment using python3 -m venv path/to/venv.
    Then use path/to/venv/bin/python and path/to/venv/bin/pip. Make
    sure you have python3-full installed.
    
    For more information visit http://rptl.io/venv

note: If you believe this is a mistake, please contact your Python installation or OS distribution provider. You can override this, at the risk of breaking your Python installation or OS, by passing --break-system-packages.
hint: See PEP 668 for the detailed specification.
```

PIP packages are nightmare, just another evidence...

---

I started to install missing packages with `apt`, `sudo apt install python3-colorama python3-cpuinfo python3-psutil` but package `pypresence` is not in Rapbian repository, that was a show stopper...

Then I tried to run DuinoMiner in virtual enviroment and it works.

```
sudo apt update && sudo apt upgrade -y
sudo apt install -y git python3 python3-dev python3-pip python3-venv
git clone https://github.com/revoxhere/duino-coin
python3 -m venv duino-coin
cd duino-coin
source bin/activate # or just . bin/activate
# python3 -m pip install -r requirements.txt  # this command was required on other RPI...
python3 PC_Miner.py
```

To start miner latter, virtual environment has to be activated again:

```
cd duino-coin
. bin/activate
python3 PC_Miner.py
deactivate  # exit from virtual environment
```

Script to start miner in an easy way:
```
$ cat miner.sh
#!/bin/sh

[ -f bin/activate ] && . bin/activate
python3 PC_Miner.py
```

**Comment by revoxhere** [comment]
Thanks for your input. The basic docs have been updated
But about the dependencies - new versions of Python require a virtual environment to work. This needs to be addressed too by updating the docs, but I need a second to get to this.

**Comment by revoxhere** [comment]
Fixed in 60e95f9198a5f1e4a0c898317d2c4f68bb68ec38
